swagger.yaml

```
@@ -39,8 +39,90 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /sites:
+    get:
+      summary: Restituisce la lista dei cantieri esistenti.
+      description: Restituisce la lista dei cantieri esistenti.
+      parameters:
+        - in: query
+          name: offset
+          schema:
+            type: integer
+          description: Numero di segnalazioni da saltare prima di restituire i risultati.
```

Cambia segnalazioni in cantieri
```suggestion
          description: Numero di cantieri da saltare prima di restituire i risultati.
```